healthchecks/Tiltfile

```diff
@@ -0,0 +1,26 @@
+#### External Functions ####
+
+# Creates a resource that performs a health-check via cURL every second, until the cURL succeeds. Returns the resource name, so other local_resources can depend on said healthcheck.
+#
+# Params:
+#    name = Name of service you're healthchecking, e.g. "elastic-search"
+#    url = URL you want to healthcheck against, e.g. http://localhost:9200/_cat/health
+#
+# Returns:
+#    resource_name = Resource name
```

Comment:
```suggestion
#    Resource name to depend on.
```

Reply:
Reworded in healthchecks and pubsub-emulator 👍🏻